CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to the Aptos TypeScript SDK will be captured in this file. T
 
 # Unreleased
 
+- Remove MoveObject in favor of AccountAddress
```

Comment:
lets make sure we document it, for Option function argument - use AccountAddress. so that people who are not familiar with it it understand